src/collection.jl

```diff
@@ -142,3 +152,94 @@ function images end
         @yield FITS(row.path)[row.hdu]
     end
 end
+
+
+"""
+    images(f, df::DataFrame; path = nothing, save_prefix = nothing, save_suffix = nothing, save_delim = "_", ext = r"fits(\\.tar\\.gz)?"i)
+
+Applies function `f` on all ImageHDUs present in data frame and saves it in FITS file.
+
+If `path = nothing`, then save functionality does not execute. It returns an array of arrays which contains final returned values of function.
+A suffix and prefix can be added to filename of newly created files by modifying `save_suffix` and `save_prefix`, `save_delim` is used as delimiter.
+`ext` is the extension of files to be taken into consideration for applying function, by default it is set to `r"fits(\\.tar\\.gz)?"i`.
```

Comment:
We will definitely want to show some examples with this, I'm not sure whether it makes more sense to copy-paste three nearly identical examples in the docstrings or whether to relagate that to the online documentation. Perhaps both?

Another way to get around this is to put all of the filename-changing stuff into the docstring for `make_file` and then show the `CCDReduction.make_file` docstring in the docs and reference it. This removes the repitition and keeps all the relevant info in docstrings, but at the cost of having to search around a little more to get the information you need.